blocks/columns/columns.css

```diff
@@ -1,16 +1,24 @@
 .columns > div {
   display: flex;
-  flex-direction: column;
+  flex-flow: column;
+  gap: 24px;
+  align-items: stretch;
 }
 
 .columns img {
-  width: 100%;
+  max-width: 100%;
 }
 
 .columns > div > div {
   container-type: inline-size;
   width: 100%;
   order: 1;
+  padding: 5px;
```

Comment:
I think you can make this 15px, see here for example https://www.learninga-z.com/site/lp2/black-friday